Dockerfile

```diff
@@ -1,51 +1,52 @@
 ARG ELIXIR_VERSION=1.14.0
 ARG ERLANG_VERSION=25.0.4
-ARG WINDOWS_VERSION=1809
+ARG ALPINE_VERSION=3.18.0
 # See also: ERTS_VERSION in the from image below
 
-ARG BUILD_IMAGE=mbtatools/windows-elixir:$ELIXIR_VERSION-erlang-$ERLANG_VERSION-windows-$WINDOWS_VERSION
-ARG FROM_IMAGE=mcr.microsoft.com/windows/servercore:$WINDOWS_VERSION
-
-FROM $BUILD_IMAGE as build
+FROM hexpm/elixir:${ELIXIR_VERSION}-erlang-${ERLANG_VERSION}-alpine-${ALPINE_VERSION} as build
 
 ENV MIX_ENV=prod
 
-# log which version of Windows we're using
-RUN ver
+RUN mkdir /realtime_signs
 
-RUN mkdir C:\realtime_signs
+WORKDIR /realtime_signs
 
-WORKDIR C:\\realtime_signs
+RUN apk add --no-cache git
+RUN mix local.hex --force && mix local.rebar --force
 
 COPY mix.exs mix.lock ./
 RUN mix deps.get
 
-COPY config/config.exs config\\config.exs
-COPY config/prod.exs config\\prod.exs
+COPY config/config.exs config/config.exs
+COPY config/prod.exs config/prod.exs
 
 RUN mix deps.compile
 
 COPY lib lib
 COPY priv priv
 
-COPY config/runtime.exs config\\runtime.exs
-RUN mix release
+COPY config/runtime.exs config/runtime.exs
+RUN mix release linux
```

Comment:
I got an error at this line; did you mean to also check in a change to `mix.exs`?

Reply:
Ah yes, thank you